src/javascripts/legacy_app.js

```diff
@@ -41,6 +42,12 @@ var App = {
 
   logClosedModal: function() {
     console.log('Closed a modal.');
+  },
+
+  storeClosedAt: function() {
+    var closedAt = new Date();
+    console.log('CLOSING', closedAt);
+    this.store('lastClosedAt', closedAt);
```

Comment:
> - the single existing test was failing
> - this repo has no CI

Yea, that's a problem, we should fix the test and setup. 

> - some examples in the code and some examples in tests means nobody knows where to look

IMHO the goal of this file is to purely demonstrate you can write your app like you did in v1, apart from a few extra details (e.g. import/export statements). The more things we add the harder it will be for people to figure that out. I think the demo apps repo is a better place to document how specific APIs work in v2. If we add additional APIs to the scaffold to make it easier to work with v2 then that should probably go in the from-scratch branch and documented separately.